frontend/storybook/src/components/entry/publish/pages/publish-one.ts

```diff
@@ -34,23 +31,19 @@ export default {
       
       }
 
-const STR_TITLE ="Settings and JIG info.";
-const STR_SUBTITLE = "Last step before publishing";
+
 const STR_BTNLABEL = "Publish JIG";
 const STR_IMGTHUMBNAIL = "red-sea-book.png";
 const STR_SLIDERLABEL = "My JIG is public";
 const STR_NAME = "JIG’s name";
 const STR_LANGUAGE = "Language of instructions";
```

Comment:
shouldn't some of these be in an element (like AGE, LANGUAGE)?

Reply:
these are used for dropdowns that are elements already